src/Extensions/ElementalAreaUsedOnTableExtension.php

```diff
@@ -3,24 +3,80 @@
 namespace DNADesign\Elemental\Extensions;
 
 use SilverStripe\Admin\Forms\UsedOnTable;
+use SilverStripe\CMS\Model\SiteTree;
 use SilverStripe\ORM\DataExtension;
 use SilverStripe\ORM\ArrayList;
 use SilverStripe\ORM\DataObject;
+use SilverStripe\ORM\ValidationException;
 use DNADesign\Elemental\Models\ElementalArea;
+use DNADesign\Elemental\Models\BaseElement;
 
 class ElementalAreaUsedOnTableExtension extends DataExtension
 {
 
     /**
      * Hides ElementalArea's from the "Used On" tab when viewing files
      *
+     * @var array $excludedClasses
+     */
+    public function updateUsageExcludedClasses(array &$excludedClasses)
+    {
+        $excludedClasses[] = ElementalArea::class;
+    }
+
+    /**
+     * Legacy function kept for semver, replaced with updateUsageExcludedClasses above
+     *
      * @return void
-     * @var ArrayList $dataObjects
+     * @var ArrayList $usage
      * @var DataObject $record
      * @see UsedOnTable::updateUsage
+     * @deprecated 4.5.0 Use self::updateUsageExcludedClasses instead
+     */
+    public function updateUsage(ArrayList &$usage, DataObject &$record)
```

Comment:
Is this truly semver? What I mean is, if someone had extended this class and implemented this method, would it still do what they expected after updating to CMS 4.7 and Elemental 4.5?

Reply:
Also - won't dropping the contents of this method change the behaviour of Used On entries for Elemental when run against CMS 4.6 and earlier?

Reply:
Resolved by raising the minimum core requirement to 4.7.